packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.types.ts

```diff
@@ -20,56 +20,53 @@ export enum YppAuthorizationErrorCode {
   CHANNEL_CRITERIA_UNMET_VIDEOS = 'CHANNEL_CRITERIA_UNMET_VIDEOS',
   CHANNEL_CRITERIA_UNMET_CREATION_DATE = 'CHANNEL_CRITERIA_UNMET_CREATION_DATE',
   YOUTUBE_QUOTA_LIMIT_EXCEEDED = 'YOUTUBE_QUOTA_LIMIT_EXCEEDED',
+  YOUTUBE_API_NOT_CONNECTED = 'YOUTUBE_API_NOT_CONNECTED',
+  QUERY_NODE_NOT_CONNECTED = 'QUERY_NODE_NOT_CONNECTED',
 }
 
-export type YppRequirementsErrorCode = Omit<
+export type YppRequirementsErrorCode = PickEnum<
   YppAuthorizationErrorCode,
-  YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED | YppAuthorizationErrorCode.CHANNEL_NOT_FOUND
+  | YppAuthorizationErrorCode.CHANNEL_CRITERIA_UNMET_SUBSCRIBERS
+  | YppAuthorizationErrorCode.CHANNEL_CRITERIA_UNMET_VIDEOS
+  | YppAuthorizationErrorCode.CHANNEL_CRITERIA_UNMET_CREATION_DATE
 >
 
 export type ChannelVerificationSuccessResponse = {
   email: string
   userId: string
 }
 
-export type ChannelRequirments = {
+export type ChannelRequirements = {
   MINIMUM_SUBSCRIBERS_COUNT: number
   MINIMUM_VIDEO_COUNT: number
   MINIMUM_VIDEO_AGE_HOURS: number
   MINIMUM_CHANNEL_AGE_HOURS: number
 }
 
-type ChannelRequirmentsFailedError = {
+type ChannelRequirementsFailedError = {
   errorCode: YppRequirementsErrorCode
   message: string
   result: number | string | Date
   expected: number | string | Date
 }
 
-export type ChannelNotFoundError = {
-  errorCode: YppAuthorizationErrorCode.CHANNEL_NOT_FOUND
+export type YppError = {
+  code: YppAuthorizationErrorCode
   message: string
 }
 
-export type ChannelAleadryRegisteredError = {
-  errorCode: YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED
-  message: string
+export type ChannelAlreadyRegisteredError = YppError & {
   // already registered channel id
+  code: YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED
```

Comment:
```suggestion
  code: YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED
    // already registered channel id
```
Tiny thing - this should be above `result`